schemas/common/descriptors/search/searchdescriptor.example.1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "@id": "https://ns.adobe.com/xdm/common/descriptors/searchdescriptor",
+  "@type": "xdm:searchdescriptor",
+  "search:indexable": true,
+  "search:fetchable": true,
+  "search:filterable": true,
+  "search:stored": false,
```

Comment:
Maybe I'm getting this example wrong, but fetchable: true and stored: false are contradicting, right? What is the expected result?